src/GRA.Controllers/MissionControl/ChallengesController.cs

```diff
@@ -435,12 +434,20 @@ public async Task<IActionResult> Edit(int id)
                 Groups = await _challengeService.GetGroupsByChallengeId(challenge.Id),
                 RelatedEvents = await _eventService.GetByChallengeIdAsync(challenge.Id),
                 BadgeMakerUrl = GetBadgeMakerUrl(siteUrl, site.FromEmailAddress),
-                UseBadgeMaker = true
+                UseBadgeMaker = true,
+                IgnorePointLimits = UserHasPermission(Permission.IgnorePointLimits),
+                MaxPointLimit = await _challengeService.GetMaximumAllowedPointsAsync(site.Id)
             };
 
-            if (site.MaxPointsPerChallengeTask.HasValue)
+            if (challenge.TasksToComplete.HasValue)
             {
-                viewModel.MaxPointsMessage = $"(Up to {site.MaxPointsPerChallengeTask.Value} points per required task)";
+                viewModel.MaxPointLimit = await _challengeService.GetMaximumAllowedPointsAsync(site.Id);
```

Comment:
This value is already set in the view model above.

Reply:
Resolved in commit f7dc1868f3ce536a6ed61fe20809c011e0ea7ae6